base/event.jl

```diff
@@ -447,11 +450,32 @@ end
 """
     Timer(callback::Function, delay, repeat=0)
 
-Create a timer to call the given `callback` function. The `callback` is passed one argument,
-the timer object itself. The callback will be invoked after the specified initial `delay`,
-and then repeating with the given `repeat` interval. If `repeat` is `0`, the timer is only
-triggered once. Times are in seconds. A timer is stopped and has its resources freed by
-calling [`close`](@ref) on it.
+Create a timer that wakes up tasks waiting for it (by calling [`wait`](@ref) on the timer object) and 
+calls the function `callback`. 
+
+Waiting tasks are woken and the function `callback` is called after an intial delay of `delay` seconds, 
+and then repeating with the given `repeat` interval in seconds. If `repeat` is equal to `0`, the timer 
+is only triggered once. The function `callback` is called with a single argument, the timer itself. 
+When the timer is closed (by [`close`](@ref) waiting tasks are woken with an error. Use [`isopen`](@ref) 
+to check whether a timer is still active.
+
+# Examples
+
+Here the first number is printed after a delay of two seconds, then the following numbers are printed quickly. 
+
+```julia-repl
+julia> begin
+       i=0
+       cb(timer) = println(global i +=1)
+       t=Timer(cb, 2,0.2)
+       wait(t)
+       sleep(.5)
+       close(t)
+       end
```

Comment:
Typically we use the preferred style even in REPL examples. So this should be written:
```
julia> begin
           i = 0
           cb(timer) = (global i += 1; println(i))
           t = Timer(cb, 2, 0.2)
           wait(t)
           sleep(0.5)
           close(t)
       end
```